winml/lib/Api/LearningModel.cpp

```diff
@@ -22,24 +20,19 @@ WINML_CATCH_ALL
 LearningModel::LearningModel(
     const std::string& path,
     const winml::ILearningModelOperatorProvider operator_provider) try : operator_provider_(operator_provider) {
-  _winmlt::PerformanceTelemetryEvent kLoadModel_event(
-      WinMLRuntimePerf::kLoadModel);
 
   WINML_THROW_IF_FAILED(OrtGetWinMLAdapter(adapter_.put()));
   WINML_THROW_IF_FAILED(adapter_->OverrideSchemaInferenceFunctions());
   WINML_THROW_IF_FAILED(adapter_->CreateModelProto(path.c_str(), model_proto_.put()));
 
   Initialize();
 
-  LogCreationEvent(true);
 }
 WINML_CATCH_ALL
 
 LearningModel::LearningModel(
     const wss::IRandomAccessStreamReference stream,
     const winml::ILearningModelOperatorProvider operator_provider) try : operator_provider_(operator_provider) {
-  _winmlt::PerformanceTelemetryEvent kLoadModel_event(
-      WinMLRuntimePerf::kLoadModel);
```

Comment:
This event doesn't send up telemetry. It merely logs a runtime ETW event. This is because there is no telemetry keyword in the TraceLoggingWrite call. An example of telemetry keyword is: MICROSOFT_KEYWORD_MEASURES, MICROSOFT_KEYWORD_TELEMETRY, or MICROSOFT_KEYWORD_CRITICAL_DATA.

I think it would be useful to still have a ETW tracelogging write event that indicates that a load model event in WinML.